server/metrics/collectors/collectors.go

```diff
@@ -16,37 +16,37 @@ const (
 
 	subsystemPrefix = Subsystem + "_"
 
-	// Operations is the key for CRI-O operation metrics.
+	// Operations is the key for CRI-O operation metrics. (to be deprecated)
```

Comment:
Let's add in which favor it has been deprecated.

Reply:
Added in b1a1bfa.